src/main/kotlin/org/opensearch/commons/replication/action/ReplicationActions.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.commons.replication.action
+
+import org.opensearch.action.ActionType
+import org.opensearch.action.support.master.AcknowledgedResponse
+
+/**
+ * All the transport action information for the Replication plugin
+ */
+object ReplicationActions {
+
+    /**
+     * Stop replication. Internal only - Inter plugin communication.
+     */
+    const val STOP_REPLICATION_NAME = "indices:admin/plugins/replication/index/stop"
+    const val STOP_REPLICATION_BASE_ACTION_NAME = "indices:admin/plugins/replication/index/unfollow"
+
+    /**
+     * Stop replication transport action type. Internal only - Inter plugin communication.
+     */
+    val STOP_REPLICATION_ACTION_TYPE =

```

Comment:
Can we define different ActionType for base and stop replication action? So from plugin, we would invoke the TransportAction using the ActionType of base action and requires both 2 permissions. While for the REST API, everything is kept like before.

Reply:
> Can we define different ActionType for base and stop replication action? So from plugin, we would invoke the TransportAction using the ActionType of base action and requires both 2 permissions. While for the REST API, everything is kept like before.

Yes, thanks for the suggestion @bowenlan-amzn. I've done the same now. 
Added UNFOLLOW_REPLICATION_ACTION_TYPE for plugin, while STOP_REPLICATION_ACTION_TYPE would be used for REST API.